wordpress-mobile/WordPress-iOS
# Clean up Gutenberg related code.

This is a simple clean up of code that was part of the "experimental" phase.

- Removed commented/unnecessary code.
- Renamed GutenbergController to GutenbergViewController.
- Moved the Gutenberg subfolder to `Classes/ViewRelated`.
- Moved `PostEditorNavigationBarManager` to its own file.

Some of the clean up made it possible to open Pages with Gutenberg.

**Note:**
There's still work left to do about `MediaProvider.swift`.
That will be done on the next PR.

To test:
- Checkout this branch and remember to switch ON the gutenberg flag. (#10497)
- Check that it builds and run successfully.
- Check that Post and Pages are opened with Gutenberg.

cc @koke
